src/cruise_control/scripts/cruise_control.py

```diff
@@ -38,11 +41,14 @@ def listener(self):
         # Higher the rate the smoother the constant speed is, but will take more CPU power
         rate = rospy.Rate(30)
         while not rospy.is_shutdown():
-         self.publish_results()
-         rate.sleep()
+            self.publish_results()
+            rate.sleep()
 
-    def handle_lidar_data(self, data):
-        print("Obtained lidar data")
+    def handle_lidar_data(self, data: PointCloud):
+        # Bounding boxes are sent as a flat array
+        float_array = Float64MultiArray()
```

Comment:
Should make a custom message for this, but when I try to do catkin_make it doesn't detect my new messages, not sure what's up with that

Reply:
You can check Andrew's or my PR on what changes you need to make in the CMakeLists.txt file. I think there's a couple of things to add.

Reply:
Going to do this in another PR